UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API.FunctionalTests/Model/File.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace UKHO.ExternalNotificationService.API.FunctionalTests.Model
+{
+    public class File
```

Comment:
As commented above. Please rename class something similar to BatchFile instead of File

Reply:
done